dev/test/transaction.ts

```diff
@@ -457,21 +459,6 @@ describe('failed transactions', () => {
     ).to.eventually.be.rejectedWith('Final exception');
   });
 
-  it('fails on beginTransaction', () => {
```

Comment:
This test no longer works, since the retries are handled in google-gax, which is replaced in the unit tests with dumb implementation that doesn't retry.